### DISCUSSION TITLE
No file stored, no error

### DISCUSSION THREAD

**Discussion by nakarimi** [spatie/laravel-backup#1520] [General] [answered]
I configured the backup on the ubuntu system, it is working fine with the command line and in the controller.
I transferred it to windows, set the binary path, and it is working fine and made a backup by command line, but when using a controller with a route to creating a backup it is not working. No error returned, but no backup file was created as well.

I run the project with PHP artisan serve.

```
  public function newBackup(Request $request){
    try {
      Artisan::call("backup:run");
    } catch (\Throwable $th) {
      throw $th;
    }
  }

```

**Comment by freekmurze** [comment, marked as answer]
As mentioned in our [requirements](https://spatie.be/docs/laravel-backup/v8/requirements), we don't support Windows fully. 

Should you find why creating a backup in a controller isn't working on Windows, I'd appreciate a PR that fixes it (if the code is not too complicated).

**Reply by creativitycomkw** [reply]
i am checking on macbook, its not working at all

its working in terminal but its not working in controller
